jboss-eap72-openshift/added/standalone-openshift.xml

```diff
@@ -430,14 +430,16 @@
                     <protocol type="UNICAST3"/>
                     <protocol type="pbcast.STABLE"/>
                     <!-- ##JGROUPS_AUTH## -->
-                    <protocol type="pbcast.GMS"/>
+                    <protocol type="pbcast.GMS">
+                        <property name="join_timeout">30000</property>
```

Comment:
We are not really using the udp stack AFAIK, but anyway, the join_timeouts should be aligned with the one for tcp, unless there are reasons for having them different.

Reply:
@maschmid Agreed, I'll align these to the same value shortly.

Reply:
Agreed.